ui/src/app/dashboard/controllers/DashCtrl.js

```diff
@@ -69,14 +69,14 @@
       });
     }
 
-    $scope.removeLink = function (id) {
+    $scope.removeLink = function (index, id) {
       // Shows window confirmation before deleting download.
       var deleteDownload = $window.confirm('Confirm download task deletion !');
 
       if(deleteDownload){
         DashService.removeDownload(id).then(function(response){
           ToastService.showToast("Download removed");
-          getActiveDownloads();},
```

Comment:
Before removing this, just check the response. If the response is ok, means HTTP 200 then remove index else you have to show that something is wrong (A `ToastService.showToast()`).
Check the function that runs on delete request.
https://github.com/scorelab/Bassa/blob/develop/components/core/routes/Download.py#L69
Btw, Nice job man

Reply:
Hey thanks for pointing that out. Applied the necessary changes.